pkg/commands/help.go

```diff
@@ -37,16 +56,21 @@ func newHelpCommand(logger logutils.Log) *helpCommand {
 		},
 	}
 
-	helpCmd.AddCommand(
-		&cobra.Command{
-			Use:               "linters",
-			Short:             "Help about linters",
-			Args:              cobra.NoArgs,
-			ValidArgsFunction: cobra.NoFileCompletions,
-			Run:               c.execute,
-			PreRunE:           c.preRunE,
-		},
-	)
+	lintersCmd := &cobra.Command{
+		Use:               "linters",
+		Short:             "Help about linters",
+		Args:              cobra.NoArgs,
+		ValidArgsFunction: cobra.NoFileCompletions,
+		RunE:              c.execute,
+		PreRunE:           c.preRunE,
+	}
+
+	helpCmd.AddCommand(lintersCmd)
+
+	fs := lintersCmd.Flags()
+	fs.SortFlags = false // sort them as they are defined here
+
+	fs.BoolVar(&c.opts.JSON, "json", true, color.GreenString("Display as JSON"))
```

Comment:
I guess, by default, `json` should be false:
```suggestion
	fs.BoolVar(&c.opts.JSON, "json", false, color.GreenString("Display as JSON"))
```

Because now the command without `--json` prints in JSON:
```
❯ go run ./cmd/golangci-lint/ help linters

[{"name":"asasalint","description":"Check for pass []any as any in variadic func(...any).","fast":false,"autoFix":false,"presets":["bugs"],"enabledByDefault":false,"deprecated":false,"since":"v1.47.0","originalURL":"https://github.com/alingse/asasalint"},{"name":"asciicheck","description":"Checks that all code identifiers does not have non-ASCII symbols in the name.","fast":true,"autoFix":false,"presets":["bu
```